interactive_templates/templates/v2/analysis/demographics.py

```diff
@@ -64,12 +64,12 @@
     "imd": (
         patients.categorised_as(
             {
-                "0": "DEFAULT",
-                "1": """index_of_multiple_deprivation >=1 AND index_of_multiple_deprivation < 32844*1/5""",
+                "Missing": "DEFAULT",
+                "Most deprived": """index_of_multiple_deprivation >=1 AND index_of_multiple_deprivation < 32844*1/5""",
```

Comment:
This should be >=0 
See [documentation](https://docs.opensafely.org/study-def-tricks/#grouping-imd-by-quintile)